### PR TITLE
Improve golint logs. Set a big postBuffer for git fetches.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

**Comment by spxtr** [comment]
/retest

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/3598/pull-test-infra-bazel/4922/
That's worrying.

File: prow/git/git.go [hunk 1]
```diff
@@ -114,6 +114,11 @@ func (c *Client) Clone(repo string) (*Repo, error) {
 		if b, err := exec.Command(c.git, "clone", "--mirror", remote, cache).CombinedOutput(); err != nil {
 			return nil, fmt.Errorf("git cache clone error: %v. output: %s", err, string(b))
 		}
+		pbs := exec.Command(c.git, "config", "http.postBuffer", "524288000")
```
Comment (kargakis) [inline prow/git/git.go]: Add a comment as to why this is needed.

**Comment by spxtr** [comment]
I'm going to do this in a different way.

**Pull request closed without merging**